src/UXClient/Components/DateTimePicker/DateTimePicker.ts

```diff
@@ -547,7 +554,8 @@ class DateTimePicker extends ChartComponent {
                 .classed("tsi-timeLabel", true)
                 .attr('id', inputLabelID)
                 .attr('for', inputID)
-                .attr('aria-label', `${startOrEnd === 'start' ? this.getString('Start time input') : this.getString('End time input')}`)
+                .attr('required',true)
+                .attr('aria-label', `${startOrEnd === 'start' ? this.getString('Start time Checking') : this.getString('End time input')}`)
```

Comment:
Why change it to 'Start time Checking'?

Reply:
This is not required. I'll remove this changes